src/store/index.js

```diff
@@ -1,4 +1,44 @@
 import { createStore } from 'vuex'
+import app from '@/helpers/axiosConfig'
+
+const pokemons = {
+  namespaced: true,
+  state: {
+    list: [],
+    pokemon: null,
+    abilities: [],
+  },
+  mutations: {
+    addPokemon(state, pokemon) {
+      const pokemons = JSON.parse(localStorage.getItem('pokemons'))
+      if(pokemons){
+        state.list = pokemons
+      }
+      state.list.unshift(pokemon)
+      localStorage.setItem('pokemons', JSON.stringify(state.list))
+    },
+    getPokemon(state, name) {
+      const pokemons = JSON.parse(localStorage.getItem('pokemons'))
+      if(pokemons){
+        state.pokemon = pokemons.find(pokemon => pokemon.name === name)
+      } else state.pokemon = state.list.find(pokemon => pokemon.name === name)
+      state.abilities = [
```

Comment:
Why do we create a separate array here instead of using the one which is already part of the pokemon object?

Reply:
Only for the scope of merging them in one array in order to loop through it in TabItem and not need to repeat the column code. Better suggestion on how to achieve that?

Reply:
Having them in an array to iterate through would be fine. But it shouldn't be named abilities as you create sections. Then inside of the objects the abilities key is misleading as it's not abilities (or not in all cases) but subsections. This resonates with my comment on the tab/tabitem components.

Reply:
If you plan to stick with the generic components here then rename the abilities key in the objects in the array to something generic as well, or create three different component with different keys to avoid confusion later on.